src/test/java/games/strategy/util/PropertyUtilTest.java

```diff
@@ -54,7 +54,7 @@ public void testGetField() {
 
   @Test
   public void testGetFieldWithPrefixedPropertyNames() {
-    assertThat(PropertyUtil.getPropertyFieldObject(BAR, new mUnderBarClass()), is(DEFAULT));
+    assertThat(PropertyUtil.getPropertyFieldObject(BAR, new UnderBarClass()), is(DEFAULT));
```

Comment:
This test looks like it's specifically verifying that the `PropertyUtil#getPropertyFieldObject()` method can find a field with an `m_` prefix.  I don't think the `UnderBarClass#m_bar` field should be renamed.  We can always suppress this warning when we introduce Checkstyle suppression filters.

However, it seems perfectly fine to have renamed `mUnderBarClass` to `UnderBarClass`.  `PropertyUtil` doesn't seem to have any dependency on the class name.